editor/plugins/animation_blend_tree_editor_plugin.h

```diff
@@ -75,7 +75,7 @@ class AnimationNodeBlendTreeEditor : public AnimationTreeNodeEditorPlugin {
 		String type;
 		Ref<Script> script;
 		int input_port_count;
-		AddOption(const String &p_name = String(), const String &p_type = String(), bool p_input_port_count = 0) :
+		AddOption(const String &p_name = String(), const String &p_type = String(), bool p_input_port_count = false) :
```

Comment:
That's actually the `bool` here which seems to be a bug. `p_input_port_count` definitely sounds like something that should be an `int`.